src/types/data/index.sass

```diff
@@ -63,19 +64,13 @@
     padding-left: 17px
 
   &.date
-    display: flex
-    border: 1px solid #cccccc
-    border-radius: 4px
-
-    > input
-      height: 100%
-      padding: 10px
-      border-radius: 4px
-      font: inherit
+    position: relative
+    padding-left: 34px
 
-    > .react-date-field__calendar-icon
-      order: -1
-      margin: 2px 2px 0 4px
+    + .tether-target
```

Comment:
You're right! Just had to update the tether props.

Also made it so the picker should flip above the input box if it's on the bottom of the page and doesn't have room to display.